lxd/device/nic_bridged.go

```diff
@@ -631,10 +631,20 @@ func (d *nicBridged) Start() (*deviceConfig.RunConfig, error) {
 	}
 
 	if d.inst.Type() == instancetype.VM {
+		mtuStr, ok := d.config["mtu"]
+		if !ok {
+			mtuInt, err := d.getHostMTU()
```

Comment:
Lets skip this call and only send mtu in the config if specified by NIC/parent network, this is consistent with how the likes of `networkCreateVethPair` and `networkCreateTap` work and avoids the need for the syscall.